archlinuxcn/oniux-git/PKGBUILD

```
@@ -0,0 +1,47 @@
+# Maintainer: Mumi Jim <echo "=02bj5yav9Gb0V3bA1Waq9VatVXb" | rev | base64 -d>
+#Thanks Integral and Klaus
+#only for x86_64
+
+pkgname="oniux-git"
+_appname="oniux"
+pkgver=0.4.0
+pkgrel=1
+url='https://gitlab.torproject.org/tpo/core/oniux'
+license=('MIT' 'Apache-2.0')
```

```suggestion
license=('MIT OR Apache-2.0')
```
Please [use SPDX license identifiers](https://rfc.archlinux.page/0016-spdx-license-identifiers)